src/routes/productRouter.js

```diff
@@ -1,9 +1,9 @@
-const express = require('express');
+const expreess = require('express');
+const productRouter = expreess.Router();
 
-const productController = require('../controlloers/productController');
-
-const productRouter = express.Router();
+const productController = require('../controllers/productController');
 
+productRouter.get('/all', productController.getAllProducts);
```

Comment:
```suggestion
productRouter.get('', productController.getAllProducts);
```
`all`은 굳이 적어주지 않아도 될 것 같아요~ 실제로 모든 product를 보여주는 것은 아니니까요. 굳이 바꾼다면 list가 있겠습니다만 그것도 굳이? 라는 생각이 드네요 ㅎㅎ